datadog/resource_datadog_dashboard.go

```diff
@@ -6180,35 +6180,63 @@ func getWidgetCustomLinkSchema() map[string]*schema.Schema {
 		"label": {
 			Description: "The label for the custom link URL.",
 			Type:        schema.TypeString,
-			Required:    true,
+			Optional:    true,
 		},
 		"link": {
 			Description: "The URL of the custom link.",
 			Type:        schema.TypeString,
-			Required:    true,
+			Optional:    true,
+		},
+		"is_hidden": {
+			Description: "The flag for toggling context menu link visibility",
+			Type:        schema.TypeBool,
+			Optional:    true,
+		},
+		"override_label": {
+			Description: "The label id that refers to a context menu link item",
+			Type:        schema.TypeString,
+			Optional:    true,
```

Comment:
Could not use `ConflictsWith` property here due to `custom_link` being in a list. 

See  https://github.com/hashicorp/terraform-plugin-sdk/issues/71